netbox/ipam/tables/vlans.py

```diff
@@ -65,9 +65,11 @@ class Meta(NetBoxTable.Meta):
         model = VLANGroup
         fields = (
             'pk', 'id', 'name', 'scope_type', 'scope', 'vid_ranges_list', 'vlan_count', 'slug', 'description',
-            'tags', 'created', 'last_updated', 'actions', 'utilization',
+            'tenant', 'tenant_group', 'tags', 'created', 'last_updated', 'actions', 'utilization',
```

Comment:
VLANGroupTable needs to inherit TenancyColumnsMixin to ensure the `tenant` and `tenant_group` columns render correctly as hyperlinks.